overlord/configstate/configcore/refresh.go

```diff
@@ -40,6 +40,19 @@ func init() {
 	supportedConfigurations["core.refresh.rate-limit"] = true
 }
 
+func reportOrResetInvalidManageRefreshes(tr config.Conf, optName string) error {
+	// check if the option is set as part of transaction changes; if not than
+	// it's already set in the config state.
+	for _, k := range tr.Changes() {
+		if k == "core."+optName {
+			return fmt.Errorf("cannot set schedule to managed")
+		}
+	}
+	tr.Set("core", optName, nil)
```

Comment:
I'm a bit unsure whethere resetting the option is what we want vs just ignoring it though. The code in refreshScheduleManaged in autorefresh.go will do anyway the right thing ignoring the setting?

Reply:
You're right; we logger.Noticef in refreshScheduleManaged. Whether this should be turned into a warning there is an open question I guess?